### PR TITLE
feat: auto-label PRs that contain db migrations

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: .github/workflows/label-miration-pr.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Label Migration PR
```
Comment (john-bodley) [inline .github/workflows/label-miration-pr.yml:1]: There's a typo in the name, `s/miration/migration`.

**Comment by john-bodley** [comment]
Thanks @mistercrunch for the PR. Would you please complete the PR description template?

**Comment by rusackas** [comment]
The PR won't run new actions from a fork, so this one, so you might have to YOLO merge the action, and then open a test PR with the stub file. 

Does this also run on changed migration files, or deleted files? Looks like it would.

**Comment by mistercrunch** [comment]
This is a bit of an experiment, should have been DRAFT, I was going to fill in the template if/when I can make it work.

**Comment by john-bodley** [comment]
BTW @mistercrunch previously CI didn't run in `DRAFT` mode, though now it does.

**Pull request closed without merging**